tmva/sofie/test/TestCustomModelsFromONNX.cxx

```diff
@@ -725,6 +728,28 @@ TEST(ONNX, AvgPool){
 
 }
 
+TEST(ONNX, Shape){
+   constexpr float TOLERANCE = DEFAULT_TOLERANCE;
+
+   // Preparing the standard  input
+   std::vector<float> input({
+      1, 2
+   });
+   
+   TMVA_SOFIE_Shape::Session s("Shape_FromONNX.dat");
+   std::vector<int> output = s.infer(input.data());
```

Comment:
The output is a vector of int64_t, but you can simply do:
`auto output = s.infer(input.data());`